### PR TITLE
isConnected query removed

### PR DESCRIPTION
**Pull request by TDCroPower**

### CODE REVIEW

**Comment by mcm1957** [comment]
Thanks for the PR.

Did you test the behavior in case no connection is available? As you do no longer check isConnected the requests most likely will fail. So please verify (if not yet done) that

-) the adapter does not crash / shutdown due to the failing poll requests
-) the ioBroker log is not flooded with error messages

Thanks in advance for testing
@Apollon77 - FYI if you have more information

@Lucky-ESA 
Looks like you fixed (or at least tried to fix) the same issue a short time ago. Could you check this PR / drop any statement please.

**Comment by TDCroPower** [comment]
@mcm1957 I have just tested my fork.
Box Standby On/Off -> works
Adapter Stop/Start -> works
Box restart -> works

**Comment by Lucky-ESA** [comment]
"isConnected" is also for deepstandby
Unnecessary requests are made.

**Comment by TDCroPower** [comment]
I have still committed the logical next version numbers.
In all tests the change was successful.
When I send my box to DeepStandby, no objects are updated and enigma2-CONNECTION is set to false.
No error logs or anything like that are seen.
Don't see any problems to release this as an update.

**Comment by Apollon77** [comment]
@TDCroPower Thank you cool work!

please remove the version increase and such and just add a "WORK IN PROGRESS" section to the readme ... the release script we use willdo all this version increase automatically.

**Comment by TDCroPower** [comment]
@Apollon77 did a revert to the version changes

File: README.md [hunk 1]
```diff
@@ -125,6 +125,9 @@ sendTo("enigma2.0", "send", {
     Placeholder for the next version (at the beginning of the line):
     ### **WORK IN PROGRESS**
 -->
+### WORK IN PROGRESS
```
Comment (Apollon77) [inline README.md]: Please really use 

### **WORK IN PROGRESS**

(with the stars) and please ideally write human readable user direct english description of the changes in short ... links (I know also done in last changelog entry) are not that good for the users and display (sorry to bother with such formal stuff)

Comment (TDCroPower) [reply README.md]: fixed it

Comment (mcm1957) [reply README.md]: Resulting changes look good

**Pull request merged**